lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -91,8 +95,8 @@
       <li class="action action-share">
         <a class="action-linkedin-profile" target="_blank" href="${cert_status['linked_in_url']}"
          title="${_('Add Certificate to LinkedIn Profile')}"
-         data-course-id="${unicode(course_overview.id)}"
-         data-certificate-mode="${cert_status['mode']}"
+         data-course-id="${course_overview.id | n, unicode}"
+         data-certificate-mode="${cert_status['mode'] | n, js_escaped_string}"
```

Comment:
Data attributes are tricky. The fourth item [here](http://draft-safe-templates-2016-03-23.readthedocs.org/en/latest/conventions/safe_templates.html#overview-of-contexts) suggests that data attributes should be HTML-escaped.